python/bifrost/udp_transmit.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from libbifrost import _bf, _check, _get
+
+import ctypes
+import numpy as np
+
+def _packet2pointer(packet):
+	buf = ctypes.c_char_p(packet)
+	siz = ctypes.c_uint( len(packet) )
+	return buf, siz
+
+
+def _packets2pointer(packets):
+	count = ctypes.c_uint( len(packets) )
+	buf = ctypes.c_char_p("".join(packets))
```

Comment:
One way to avoid overhead here would be to have `bfUdpTransmitSendMany` accept one array for the headers and a separate one for the payloads, and to use the gather capabilities of `sendmmsg` to stitch them together implicitly. This would allow passing a block of data as-is (with the headers generated as a separate string).